addons/web/static/src/core/dialog/dialog.xml

```diff
@@ -3,15 +3,15 @@
 
     <t t-name="web.Dialog" owl="1">
         <div class="o_dialog">
-            <div role="dialog" class="modal"
+            <div role="dialog" class="modal d-block"
                 tabindex="-1"
                 t-att-class="{ o_technical_modal: technical, o_modal_full: fullscreen }"
                 t-ref="modal"
                 >
                 <div class="modal-dialog" t-att-class="size">
-                    <div class="modal-content" t-att-class="contentClass">
+                    <div class="modal-content rounded-0" t-att-class="contentClass">
```

Comment:
Remove `rounded-0`... community modal-content should have a border-radius (if it's not the case fix it).
Note that by default this class uses bootstrap `$modal-content-border-radius` variable ;)